types/webpack/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for webpack 3.0
+// Type definitions for webpack 3.8
```

Comment:
can you review changes and make sure we're adding all new APIs?

Reply:
I updated some stuff that I found missing!